fe/fe-core/src/main/java/com/starrocks/connector/iceberg/ScalarOperatorToIcebergExpr.java

```diff
@@ -226,6 +227,10 @@ public Expression visitInPredicate(InPredicateOperator operator, IcebergContext
                         return literalValue;
                     }).collect(Collectors.toList());
 
+            if (literalValues.stream().anyMatch(Objects::isNull)) {
```

Comment:
It's better to add some comments here to explain this check.

Reply:
done